README.md

```diff
@@ -16,22 +16,29 @@ To gossip you need one or more seed nodes. Seed is just a list of places to init
 ```java
   GossipSettings settings = new GossipSettings();
   int seedNodes = 3;
-  List<GossipMember> startupMembers = new ArrayList<>();
+  List<Member> startupMembers = new ArrayList<>();
   for (int i = 1; i < seedNodes+1; ++i) {
     URI uri = new URI("udp://" + "127.0.0.1" + ":" + (50000 + i));
-    startupMembers.add(new RemoteGossipMember(cluster, uri, i + ""));
+    startupMembers.add(new RemoteMember(cluster, uri, i + ""));
```

Comment:
It actually may not be immediately obvious (it was not for me!) that cluster here is a String ID for the cluster.  Perhaps explicitly add this so that this code block is stand-alone runnable?

Reply:
Agree.